eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/CBSChannel.java

```diff
@@ -85,4 +50,50 @@ public void close(
 
         this.innerChannel.close(reactorDispatcher, closeCallback);
     }
+
+    private static class RequestResponseChannelOperationResult implements OperationResult<RequestResponseChannel, Exception> {
```

Comment:
Is it possible to have a base implementation for this `OperationResult<T,E>` interface? Many of these nested static classes perform the same function and are redundant.

Reply:
I took a stab at it. Not sure how I feel about it
https://github.com/mssfang/azure-sdk-for-java/compare/EventHubs-P3...conniey:eventHub